ui/app/mainui/activitycenter/panels/ContactVerificationCta.qml

```diff
@@ -0,0 +1,34 @@
+import QtQuick 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+
+import shared 1.0
+import utils 1.0
+
+StatusFlatButton {
+    id: root
+
+    property int verificationStatus: -1
+
+    signal activate()
+
+    width: implicitWidth
+    enabled: verificationStatus == Constants.verificationStatus.unverified ||
+             verificationStatus == Constants.verificationStatus.verifying
+    text: {
+        switch (verificationStatus) {
+        case Constants.verificationStatus.unverified:
+            return qsTr("Answer")
+        case Constants.verificationStatus.verifying:
+            return qsTr("Edit Answer")
+        case Constants.verificationStatus.verified:
+            return qsTr("Verification Complete")
+        default:
+            return qsTr("Unknown")
```

Comment:
```qml
    readonly property QtObject verificationStatus: QtObject {
        readonly property int unverified: 0
        readonly property int verifying: 1
        readonly property int verified: 2
        readonly property int declined: 3
        readonly property int canceled: 4
        readonly property int trusted: 5
    }
```

So when the other person is "Verified&Trusted" already it will show as "Unknown"?

Notice you can perform the ID verification also from the Profile Dialog.